### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by makus82**
here edited readme WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,9 +180,30 @@ make -j4
...
 
-## Contact us
-
-If you're interested, please visit our [website](http://openapoc.org).
-We also have [forums](http://openapoc.org/forums/) - please pop by and introduce yourself!
-We also have an IRC channel on [Freenode](http://freenode.net) - [#openapoc](irc://irc.freenode.net/#openapoc).
+* This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be acquired from [Steam](http://store.steampowered.com/app/7660/) - this is _required_ to run)
+  - you need have all files in ISO file including MUSIC etc
+  - if it's in .iso format, rename it to "cd.iso"
+  - if it's not, copy all the contents into a folder and rename the folder to "cd.iso"
```
Comment (JonnyH) [inline README.md]: We also support the .cue / .bin files (which are used, for example, in the gog.com version)

You rename the .cue file "cd.iso", put it in the data/ folder, then put the .bin file in the same folder (without changing it's name - so for example the gog.com file remails "XCOM.BIN")

That's in addition to a directory called "cd.iso" :)

File: README.md [hunk 2]
```diff
@@ -1,20 +1,105 @@
...
-All help is appreciated, but there's little point trying to 'play' it as of yet.
-
-[See our Trello TO-DO list for more info.](https://trello.com/b/lX5Y3DwR/openapoc)
-
-## Requirements
-
-* This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be acquired from [Steam](http://store.steampowered.com/app/7660/) - this is _required_ to run)
-  * If you have the real cd, you can copy all the included files to a directory called "cd.iso"
+# OpenApocalypse [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Are%20you%20fan%20X-Com%20Apocalypse?%20OpenApoc%20its%20clone%20of%20this%20great%20game%20-%20contribute!%20https://github.com/OpenApoc/OpenApoc&hashtags=games,openapoc,xcom)
+> OpenApoc is an open-source clone of the original [X-COM: Apocalypse](http://www.ufopaedia.org/index.php/Apocalypse), licensed under the MIT and written in C++ / SDL2. It was originally founded by PmProg in July 2014, and has since grown in to [community](http://www.ufopaedia.org/index.php/Credits_(OpenApoc)).
```
Comment (JonnyH) [inline README.md]: It's not really a "clone" - it's a re-implementation of the game engine.

A "Clone" means it's a standalone game, based somewhat on the original, while openapoc aims to be *exactly* the same game, and requires the original's resource files to run.

File: README.md (hunk 1 above)
Comment (makus82) [reply README.md]: Change it

**Comment by JonnyH** [comment]
Looks good - thanks :)

**Pull request merged**